### ISSUE TITLE
support for git diff --color-words

### ISSUE THREAD

**Issue by alexandreroberts** [tpope/vim-fugitive#2036]
As I mention [here on vi.SE](https://vi.stackexchange.com/questions/38286/vim-fugitive-support-for-diff-color-words), I would like to be able to use `vim-fugitive` to write `:Git diff --color-words`, by analogy with `:Git diff`, but when I do that, the output includes color codes like `^[[1m`...`^[[m` instead of actually coloring the text. 

For example:

<img width="389" alt="Screen Shot 2022-08-15 at 1 31 21 PM" src="https://user-images.githubusercontent.com/6886906/184713185-3b4b7919-5436-45af-95a9-4cfa6229a1cc.png">

In other words, as far as I can tell, `vim-fugitive` does not support the `--color-words` option of `git diff`.

I would like to request this support, especially if it is not too tricky to implement.

**Comment by tpope** [comment]
There's syntax highlighting for `:Git diff --diff-words`. That's as close as you're gonna get.

**Comment by alexandreroberts** [comment]
Great, thank you -- I hadn't realized that. I just tried it (actually `:Git diff --word-diff`), and it worked.
